misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -689,6 +709,95 @@ abstract class PathAssertionsSpec(
             expect(folder).isDirectoryFun()
         }
     }
+
+    describeFun(hasSameBinaryContentAs.name, hasSameTextualContentAs.name) {
+        val hasSameBinaryContentAsFun = hasSameBinaryContentAs.lambda
+        val hasSameTextualContentAsFun = hasSameTextualContentAs.lambda
+
+        context("has same binary content") {
+            it("${hasSameBinaryContentAs.name} - does not throw") withAndWithoutSymlink { maybeLink ->
+                val sourcePath = maybeLink.create(tempFolder.newFile("text1"))
```

Comment:
you are always testing with an empty file. IMO that is a good use case  as well  (add a `context("empty files")`) but in this context it should actually have a content.